docs/versioned_docs/version-0.3/10-home.md

```diff
@@ -5,7 +5,7 @@ slug: /
 
 :::caution
 
-Acorn is a work in progress.  Features will evolve over time and there may be breaking changes between releases.  Please give us your feedback in [Slack](https://slack.acorn.io), [Discussions](https://github.com/acorn-io/acorn/discussions), or [Issues](https://github.com/acorn-io/acorn/issues)!
+Acorn is a work in progress.  Features will evolve over time and there may be breaking changes between releases.  Please give us your feedback in [Slack](https://slack.acorn.io), [Discussions](https://github.com/acorn-io/runtime/discussions), or [Issues](https://github.com/acorn-io/runtime/issues)!
```

Comment:
Do we want to update docs for older releases? With the redirects, they shouldn't break.

Reply:
I am open to not doing it. I also thought that the updating it wouldn't break anything either.

Do you have a preference for not doing this?

Reply:
I did, but on second thought, we'll eventually remove these, and I don't expect users to be poking around in such old docs anyway.

So, I'm fine with merging as-is, old doc changes included.